Fix/379 button design in the home page

MandeepPaul opened the pull request:
## Describe your changes

Briefly describe the changes you made and their purpose.

## Issue number

#379 

## Please ensure all items are checked off before requesting a review:

- [ ] I deployed the code locally.
- [ ] I have performed a self-review of my code.
- [ ] I have included the issue # in the PR.
- [ ] I have labelled the PR correctly.
- [ ] The issue I am working on is assigned to me.
- [ ] I didn't use any hardcoded values (otherwise it will not scale, and will make it difficult to maintain consistency across the application).
- [ ] I made sure font sizes, color choices etc are all referenced from the theme.
- [ ] My PR is granular and targeted to one specific feature.
- [ ] I took a screenshot or a video and attached to this PR if there is a UI change.

## Preview
![Screenshot 2024-12-18 071641](https://github.com/user-attachments/assets/dfd31c42-e69e-44eb-bd43-b8f5d85e1220)

gorkem-bwl commented:
Thanks. A few notes.

- Can you check the radiuses of the boxes on top and at the bottom? They are not the same.
- Wording should be same as in Figma (e.g instead of hint views, we have banner views). 
- Make sure the fonts reflect the same color in Figma
- If the mouse is hovered on a card, does it look like the one in "Create a new banner"?

MandeepPaul commented:
My bad! I initially thought I only needed to update the buttons, but I have now updated the StatisticCards as well. Please find the attachment - the **'Create a a new banner'** button shows the hover state in it.

![image](https://github.com/user-attachments/assets/b385a19f-17d3-4dd1-9432-7493473d18e8)

erenfn commented:
Can you disable formatting for all files, and just enable it for the current file?

erenfn commented:
It would be great if you can revert the last commit (I believe that's where all files are formatted) and just update the relevant files

MandeepPaul commented:
Should be good now!

![image](https://github.com/user-attachments/assets/b385a19f-17d3-4dd1-9432-7493473d18e8)

gorkem-bwl commented:
Hi Mandeep. I couldn't find an answer/ref to the question below. Can you let me know? 

> If the mouse is hovered on a card, does it look like the one in "Create a new banner"?

MandeepPaul commented on frontend/src/scenes/dashboard/HomePageComponents/CreateActivityButton/CreateActivityButton.jsx:
```diff
@@ -1,33 +1,49 @@
+    <Button
+      variant="text"
+      onClick={onClick}
+      sx={{
```
copy! I will update in next commit

MandeepPaul commented:
> Yes Gorkhem, the middle card in the screenshot attached is in hover state.  
>> Hi Mandeep. I couldn't find an answer/ref to the question below. Can you let me know?
> > > If the mouse is hovered on a card, does it look like the one in "Create a new banner"?

gorkem-bwl commented:
> > Yes Gorkhem, the middle card in the screenshot attached is in hover state.
> > > Hi Mandeep. I couldn't find an answer/ref to the question below. Can you let me know?
> > > > If the mouse is hovered on a card, does it look like the one in "Create a new banner"?

Thank you.

MandeepPaul commented:
I have rewritten the commit history and removed unnecessary formatted files. Please refer to the attachment for the updated UI, where the 'Create a New Helper Link' button now shows the hover state.

## Preview 
![onboarding-ui-update](https://github.com/user-attachments/assets/7cc861f2-c425-45bf-83a1-c0cfc61afe20)

gorkem-bwl commented:
UI wise that looks good!

erenfn commented on frontend/src/scenes/dashboard/Dashboard.jsx:
```diff
@@ -51,18 +53,42 @@ const Dashboard = ({ name }) => {
     {
+      skeletonType: (
+        <BaseSkeleton
+          frontSkeletonProps={{
```
Hey, thanks for removing the code formatting for other files. The UI looks great! The only refactoring we could make here is to move these styles into a new file within the Skeletons folder and import them into `BaseSkeleton.jsx.`

An example usage of this might be:
` <BaseSkeleton guideType='popup'>`

MandeepPaul replied:
Great suggestion, I just updated it!!

erenfn reviewed (dismissed):
looks good!

erenfn commented:
@coderabbit resolve
